easybuild/easyconfigs/p/Perl/Perl-5.36.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,86 @@
+name = 'Perl'
+version = '5.36.1'
+
+homepage = 'https://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language
+
+Includes a small selection of extra CPAN packages for core functionality.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.cpan.org/src/%(version_major)s.0']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['68203665d8ece02988fc77dc92fccbb297a83a4bb4b8d07558442f978da54cc1']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('zlib', '1.2.13'),
```

Comment:
`DB` dep is missing here (required for `DB::File` which is part of standard Perl)

Reply:
I added it in #11451, but that does not clearly say where I was seeing the failure. I recall it being a Bio package, but not which one.

Reply:
#17970